src/components/application_manager/test/policy_handler_test.cc

```diff
@@ -1703,27 +1703,25 @@ TEST_F(PolicyHandlerTest, DISABLED_OnSnapshotCreated_UrlAdded) {
 
 #ifdef PROPRIETARY_MODE
   ExtendedPolicyExpectations();
-#else
-  AppIdURL next_app_url = std::make_pair(0, 0);
+#else   // HTTP mode
   EXPECT_CALL(*mock_policy_manager_, GetUpdateUrls("0x07", _))
       .WillRepeatedly(SetArgReferee<1>(test_data));
-  EXPECT_CALL(*mock_policy_manager_, GetNextUpdateUrl(_))
-      .WillOnce(Return(next_app_url));
-  EXPECT_CALL(app_manager_, application_by_policy_id(_))
-      .WillOnce(Return(mock_app));
+
+  // Check expectations for get app id
+  GetAppIDForSending();
```

Comment:
@ZhdanovP This method should return value, shouldn't it?

Reply:
@AByzhynar  No, this is the test method which defined on line 1916